### ISSUE TITLE
Failed to publish @std/expect to local jsr

### ISSUE THREAD

**Issue by jollytoad** [denoland/deno_std#4448]
**Describe the bug**

**Steps to Reproduce**

1. Install local JSR
2. Publish deno_std to local JSR as described in JSR docs
3. Fails to publish @std/expect...

```
error: Failed to publish @std/expect@0.218.2

Caused by:
    Failed to publish @std/expect at 0.218.2: failed to build module graph: Module not found "file:///_custom_equality_tester.ts".
```

**Comment by iuioiua** [comment]
This is being fixed in #4449.

**Comment by kt3k** [comment]
This now should be fixed by #4449 

(We were able to publish 0.219.1 to jsr from main) https://github.com/denoland/deno_std/actions/runs/8201450075/job/22430170040

**Comment by jollytoad** [comment]
Excellent, thank you, I can confirm publishing locally works perfectly now 👍
